woocommerce-yotpo/wc_yotpo.php

```diff
@@ -328,6 +328,37 @@ function wc_yotpo_past_order_time_query( $where = '' ) {
 }
 
 function wc_yotpo_send_past_orders() {
+    $yotpo_settings = get_option('yotpo_settings', wc_yotpo_get_degault_settings());
```

Comment:
a bit long function, is it possible to divide this function to 2-3 shorter functions? 
e.g. move block inside 'foreach ($past_orders as $post_bulk)' to separate function

Reply:
This has been the legacy code, we didnt make many changes here. But if you want it to be refactored then please create a ticket for that.